spyt-package/src/main/python/spyt/utils.py

```diff
@@ -218,8 +218,8 @@ def scala_buffer_to_list(buffer):
     return [buffer.apply(i) for i in range(buffer.length())]
 
 
-def default_user():
-    return os.getenv("YT_USER") or getpass.getuser()
```

Comment:
`import getpass` on the line 2 is no longer needed, so it also should be removed